.github/drafts/16506/16506_category_slider_draft.md

```diff
@@ -0,0 +1,38 @@
+*** Category Slider ***
+
+The Category Slider widget allows users to choose from a fixed set of incremental values.
+
+# Properties
+
+Properties allow you to edit the Category Slider widget, connect it with other widgets and customize the user actions.
+
+| **Property** | **Type** | **Description** | **Code Snippet** |
+|--------------|----------|-----------------|------------------|
+| **Options** | Formatting | An _array of objects_ with keys `value` and `label` which define the possible selections that the user can make with the slider. | `{{<widget-name>.options}}` |
+| **Default Value** | Formatting | This sets the slider's initial value, before the user makes any adjustments. This value should be between the slider's **Min Value** and **Max Value**. _(number)_ | `{{<widget-name>.defaultValue}}` |
```

Comment:
@yaldram why are these values for code snippet there for property pane items? Are you also exposing them in binding?

Reply:
No, we are not exposing the `defaultValue` as a binding. @spark-c please remove `{{<widget-name>.defaultValue}}`